apps/costcontrol/style/views/balance.css

```diff
@@ -11,15 +11,22 @@
   margin-bottom: 1rem;
   padding: 0 0 0.8rem;
   overflow: hidden;
+  display: flex;
+  align-items: center;
 }
 
 #balance-tab .overview .info {
-  transform: translateX(-4.0rem);
+  transform: translateX(calc(var(--writing-mode-offset-x-sign) * 4.0rem));
```

Comment:
Yeah, this might be too "clever" and I don't think there's any concensus on the preferred solution in this case. I'll re-work this part of the patch to use [dir="ltr|rtl"] in the selectors instead